pkgs/servers/http/unit/default.nix

```diff
@@ -1,11 +1,18 @@
 { stdenv, fetchFromGitHub, which
-, withPython ? true, python
-, withPHP72 ? true, php72
-, withPHP73 ? false, php73
-, withPerl ? true, perl
+, ncurses
+, withPython27 ? true, python27
```

Comment:
python2 is EOL in 6 weeks, i would recommend using python37 if it's supported

Reply:
Thanks, fixed.